mackup/main.py

```diff
@@ -36,6 +36,10 @@ def main():
     mckp = Mackup()
     app_db = ApplicationsDatabase()
 
+    # If we want to answer mackup with "yes" for each question
+    if args.force:
+        utils.raw_input = lambda *args: "Yes"
```

Comment:
I find this a little heavy =) Can you just check for the force argument in the utils.confirm function instead?